decoration.sk

```diff
@@ -331,3 +323,89 @@ update aquatic:
+		north-northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:7}
+		north[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:8}
+		north-northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:9}
+		northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:10}
+		east-northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:11}
+		east[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:12}
+		east-southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:13}
+		southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:14}
+		south-southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:15}
+		
+old orientations:
+	minecraft version = 1.12.2 or older
+	{skull types}:
+		skelet(on|al) (skull|head) = - {SkullType:0} 
+		wither skle(on|al) (skull|head) = - {SkullType:1}
+		zombi(e|fied) = - {SkullType:2}
+		player (head|skull) = - {SkullType:3}
+		creeper (head|skull) = - {SkullType:4}
+	{wall orientations}:
+		north[ward[s]][( |-)(facing|pointing)] = - {Damage:3}
+		south[ward[s]][( |-)(facing|pointing)] = - {Damage:2}
+		east[ward[s]][( |-)(facing|pointing)] = - {Damage:4}
+		west[ward[s]][( |-)(facing|pointing)] = - {Damage:5}
+	
+combat update skulls:
+	minecraft version = 1.9 or newer
+	[ender[ ]]dragon (head|skull) item¦s = minecraft:skull {SkullType:5}
+	#skulls on walls and floors
+	({wall orientations}|{skull floor orientations}) [ender[ ]]dragon (head|skull) item¦s = minecraft:skull {SkullType:5}
+	
+directionals before flattening:
+	minecraft version = 1.12.2 or older
+	{skull types} item¦s = minecraft:skull
+	#skulls on walls and floors
+	({wall orientations}|{skull floor orientations}) {skull types} = minecraft:skull
+	#BANNERS
+	{wall orientations} {opposite colored} wall banner = minecraft:wall_banner
+	{banner standing orientations} {opposite colored} [standing] banner¦s = minecraft:banner
+	{opposite colored} banner item¦s = -banner
+	
+directionals after flattening:
+	minecraft version = 1.13 or newer
+	skelet(on|al) (skull|head) item¦s = minecraft:skeleton_skull
+	wither skle(on|al) (skull|head) item¦s = minecraft:wither_skeleton_skull
+	zombi(e|fied) (head|skull) item¦s = minecraft:zombie_head
+	player (head|skull) item¦s = minecraft:player_head
+	creeper (head|skull) item¦s = minecraft:creeper_head
+	[ender[ ]]dragon (head|skull) item¦s = minecraft:dragon_head
+	#skulls on walls and floors
+	({directional}|{skulls floor orientations}) skelet(on|al) (skull|head) = minecraft:skeleton_wall_skull
+	({directional}|{skulls floor orientations}) wither skle(on|al) (skull|head) = minecraft:wither_skeleton_wall_skull
+	({directional}|{skulls floor orientations}) zombi(e|fied) (head|skull) = minecraft:zombie_wall_head
+	({directional}|{skulls floor orientations}) player (head|skull) = minecraft:player_wall_head
+	({directional}|{skulls floor orientations}) creeper (head|skull) = minecraft:creeper_wall_head
+	({directional}|{skulls floor orientations})[ender[ ]]dragon (head|skull) = minecraft:dragon_wall_head
```

Comment:
Shouldn't it be that the ones above (the floor skulls) have `{skull floor orientations}` while these wall skulls have just `{directional}`? Skull floor orientations shouldn't apply to ones on the wall, right? Basically, the same way you did it for banners below (wall = directional, standing = floor orientations).